src/ansys/acp/core/mechanical_integration_helpers.py

```diff
@@ -87,8 +87,8 @@ def import_acp_solid_mesh(*, mechanical: "pymechanical.Mechanical", cdb_path: PA
     """
     cdb_path = pathlib.Path(cdb_path)
 
-    if cdb_path.suffix != ".cdb":
-        raise ValueError(f"The CDB file extension must be '.cdb', not '{cdb_path.suffix}'.")
+    # if cdb_path.suffix != ".cdb":
```

Comment:
Why is that commented out? Or can it just be removed?

Reply:
FMPOV, it should be extended to also allow `.dat`.